src/test/java/org/broadinstitute/consent/http/service/dao/DatasetServiceDAOTest.java

```diff
@@ -66,26 +72,27 @@ void testInsertDatasets() throws Exception {
     User user = createUser();
 
     DatasetProperty prop1 = new DatasetProperty();
-    prop1.setSchemaProperty(RandomStringUtils.randomAlphabetic(10));
-    prop1.setPropertyName(RandomStringUtils.randomAlphabetic(10));
+    prop1.setSchemaProperty(RandomStringUtils.secure().nextAlphabetic(10));
+    prop1.setPropertyName(RandomStringUtils.secure().nextAlphabetic(10));
     prop1.setPropertyType(PropertyType.Number);
     prop1.setPropertyValue(new Random().nextInt());
 
     DatasetProperty prop2 = new DatasetProperty();
-    prop2.setSchemaProperty(RandomStringUtils.randomAlphabetic(10));
-    prop2.setPropertyName(RandomStringUtils.randomAlphabetic(10));
+    prop2.setSchemaProperty(RandomStringUtils.secure().nextAlphabetic(10));
+    prop2.setPropertyName(RandomStringUtils.secure().nextAlphabetic(10));
     prop2.setPropertyType(PropertyType.Date);
     prop2.setPropertyValueAsString("2000-10-20");
 
     FileStorageObject file1 = new FileStorageObject();
-    file1.setMediaType(RandomStringUtils.randomAlphabetic(20));
+    file1.setMediaType(RandomStringUtils.secure().nextAlphabetic(20));
     file1.setCategory(FileCategory.NIH_INSTITUTIONAL_CERTIFICATION);
     file1.setBlobId(
-        BlobId.of(RandomStringUtils.randomAlphabetic(10), RandomStringUtils.randomAlphabetic(10)));
-    file1.setFileName(RandomStringUtils.randomAlphabetic(10));
+        BlobId.of(RandomStringUtils.secure().nextAlphabetic(10),
+            RandomStringUtils.secure().nextAlphabetic(10)));
+    file1.setFileName(RandomStringUtils.secure().nextAlphabetic(10));
 
     DatasetServiceDAO.DatasetInsert insert = new DatasetServiceDAO.DatasetInsert(
-        RandomStringUtils.randomAlphabetic(20),
+        RandomStringUtils.secure().nextAlphabetic(20),
```

Comment:
Should we create roll `.secure().nextAlphabetic` into a single RandomStringUtils method?

Reply:
> Should we create roll `.secure().nextAlphabetic()` into a single RandomStringUtils method?

I'm not sure I understand the suggestion.

Reply:
Instead of calling `.secure().nextAlphabetic()` each time, have a function that calls that internally.